app/models/post.rb

```diff
@@ -2,4 +2,6 @@ class Post < ApplicationRecord
   belongs_to :user
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+    scope :update_posts_counter, ->(user, count) { user.update(posts_counter: count) }
+  scope :last_five_comments, ->(post) { Comment.where(post:).last(5) }
```

Comment:
- [ ] According to the guidelines you should have a method that updates the posts counter for a user. You can check out the guide [Scope vs Class Methods](https://www.square63.com/scope-vs-class-methods/)

```ruby
def update_post_counter
  author.increment!(:posts_counter)
end
```

- [ ] [OPTIONAL] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private

```ruby
after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```